scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -585,6 +585,8 @@ public final synchronized void start() throws IOException {
 	 * Re-starts the connector binding to the same IP address and port as
 	 * on the previous start.
 	 * 
+	 * Note: intended for unit tests only!
+	 * 
```

Comment:
Why is it for unit tests only ?

Reply:
It's package scope and final. So my feeling is, it's only usable for unit test.

Reply:
If you want to change it, feel free.
I would like to include this change in the 2.0.0RC2 tomorrow.
Therefore I merge now this PR.

Reply:
Uups sorry to not reply about that.

You can merge this !